src/release.js

```diff
@@ -219,7 +219,15 @@ async function release({
 }
 
 async function push({ cwd }) {
-  let remoteUrl = (await execa('git', ['config', '--get', 'remote.origin.url'], { cwd })).stdout;
+  let remoteUrl;
+
+  let REPOSITORY_URL = process.env.REPOSITORY_URL;
```

Comment:
Why are you using an env var and not a passed in arg?

Reply:
I can pass arg if its preferred

Reply:
Yes an arg coming from bin/commands/release.js would be preferred.